README.md

```diff
@@ -965,15 +965,21 @@ not persisted between runs.
 
 ## Parser cache
 
-A cache of parsed Puppet modules and manifests can be created to skip the use
+One or more caches of parsed Puppet modules and manifests can be created to skip the use
```

Comment:
A quick word about the merging strategy might be useful. Is it a deep merge of just the first module that's found? What has precedence, highest or lowest in the list?